clusterbuster

```diff
@@ -2410,11 +2498,14 @@ function create_standard_deployment() {
     if (($# != 5)) ; then
 	fatal "Usage: create_standard_deployment <args> namespace instance secret_count replicas containers"
     fi
-    
+
     case "${deployment_type,,}" in
 	pod)
 	    create_pod_deployment "$@"
 	    ;;
+        vm)
+            create_vm_deployment "$@"
+            ;;
```

Comment:
Put this after replicaset so that all of the cases are in the same order.

Reply:
moved it after replicaset